charts/zenko/templates/_helpers.tpl

```diff
@@ -14,3 +14,12 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- $name := default .Chart.Name .Values.nameOverride -}}
 {{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
+
+{{/*
+Create the default url for the backbeat quorum service
+*/}}
+{{- define "zookeeper.url" -}}
+{{- $zookeeperConnect := printf "%s-%s:2181" .Release.Name .Values.zookeeper.url }}
```

Comment:
Trying to get the port from .Values.backbeat-queue.zookeeper.port has issues because of the '-' in the name. Trying to use the index function as suggested [here](https://github.com/kubernetes/helm/issues/2192), however I get a 'error calling index: index of nil pointer' error with code ```{{- $port := index .Values "backbeat-queue" "zookeeper" "port" -}}```. It seems that for some reason it's not recognizing the values.

Any suggestions?

Reply:
```php
{{- $zookeeperConnect := printf "%s-%s:%d" .Release.Name .Values.zookeeper.url (int64 .Values.zookeeper.port) }}
```